.github/workflows/mutation.yml

```diff
@@ -37,11 +37,11 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-maven-
       - name: Maven Install
-        run: mvn clean install -DskipTests=true
+        run: mvn clean install -DskipTests=true --no-transfer-progress
         env:
           MAVEN_OPTS: "-Xmx1g"
       - name: Mutation Test
-        run: mvn eu.stamp-project:pitmp-maven-plugin:run -DwithHistory -DtimeoutConstant=8000
+        run: mvn eu.stamp-project:pitmp-maven-plugin:run -DwithHistory -DtimeoutConstant=8000 --no-transfer-progress
```

Comment:
Shouldn't these calls to maven use `./mvnw`? I realize that the calls were already that way, but since you added the flag `--no-transfer-progress` this no relies on the floating version of maven that ships with github action runners.